server/routes/api/documents.test.js

```diff
@@ -1230,7 +1230,11 @@ describe("#documents.search", () => {
   });
 
   it("should return documents for a specific private collection", async () => {
-    const { user, collection } = await seed();
+    const user = await buildUser();
+    const collection = await buildCollection({
+      teamId: user.teamId,
+    });
+
```

Comment:
tried to move towards model-specific builders for tests to possibly solve deadlock problems; leaving them in as it's a general improvement

Reply:
Yea, I've been trying to remove `seed` gradually to make things faster too